test/ngResource/resourceSpec.js

```diff
@@ -2101,6 +2101,28 @@ describe('cancelling requests', function() {
 
     expect(creditCard.$cancelRequest).toBe(noop);
   });
+
+  it('should not break when calling old `$cancelRequest` after the response arrives', function() {
+    $httpBackend.whenGET('/CreditCard').respond({});
+
+    var CreditCard = $resource('/CreditCard', {}, {
+      get: {
+        method: 'GET',
+        cancellable: true
+      }
+    });
+
+    var creditCard = CreditCard.get();
+    var cancelRequest = creditCard.$cancelRequest;
+
+    creditCard.$promise.then(function(list) {
+        cancelRequest();
```

Comment:
This was the piece of code mentioned in the Issue #16016 in the codepen to reproduce this issue. I suppose this is not needed since we'll be calling our function using the expect function?